test/fixtures/api/responses/get-transaction-trust-set.json

```diff
@@ -0,0 +1,32 @@
+{
+  "type": "trustline",
+  "address": "r9UHu5CWni1qRY7Q4CfFZLGvXo2pGQy96b",
+  "sequence": 449,
+  "id": "635A0769BD94710A1F6A76CDE65A3BC661B20B798807D1BBBDADCEA26420538D",
+  "specification": {
+    "limit": "10000",
+    "currency": "USD",
+    "counterparty": "rMH4UxPrbuMa1spCBR98hLLyNJp4d8p4tM",
+    "qualityIn": 500000000,
+    "qualityOut": 500000000,
+    "disableRippling": true,
+    "frozen": false,
+    "authorized": false
```

Comment:
I'm not exactly sure what the overarching rules are yet, but I see code like this elsewhere:

```
src/api/ledger/parse/payment.js:44:    allowPartialPayment: isPartialPayment(tx) || undefined,
src/api/ledger/parse/payment.js:45:    noDirectRipple: isNoDirectRipple(tx) || undefined
```